### PR TITLE
Multi-Lingual-Patch3

### PR DESCRIPTION
**Pull request by Bsharma4**
Whole platform can change language with click of a button (English -  Irish)

### CODE REVIEW

**Comment by Bsharma4** [comment]
Have deleted extra 5 files as you requested.
PS: when you work with Github desktop these files are created in your system along with the orginal repository to keep track of changes you are making.

File: index.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html/>
-<html>
+<html lang=en>
```
Comment (MarshallOkafor) [inline index.html:2]: Great! Here I can see that you added a lang tag to the HTML site.

File: index.html [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
         <!--The website header contents and links-->
         <head>
                 <title>Group4spellchecker</title>
+				<meta charset="UTF-8">
```
Comment (MarshallOkafor) [inline index.html:6]: I can see that you added a Character set information. Great!

File: index.html [hunk 3]
```diff
@@ -22,40 +23,78 @@ <h1>G4 Spell Checker Machine</h1>
                         <!-- The menu to use to manipulate the HTML page -->
                         <div class="whole">
                                 <div class="menu">
-                                        <a href="#" class = "button">New document/Doiciméad nua</a>
-                                        <a href="#" class = "button">Upload file/Uaslódáil comhad</a>
-                                        <a href="#" class = "button">Download file/Íoslódáil an comhad</a>
-                                        <a href="#" class = "button">Print/Priontáil</a>
+                                        <a id = 'nd'  class = "button">New Document</a>
```
Comment (MarshallOkafor) [inline index.html:26]: Okay, I can see that you added IDs to the link tags. Awesome!

File: index.html [hunk 4]
```diff
@@ -22,40 +23,78 @@ <h1>G4 Spell Checker Machine</h1>
...
                         </div>
         
                         <!-- The footer of the HTML page -->
-                        <div class="footer">© copyright Group-4 CSCI-5030 2021</div>
+                        <div class="footer" id ='ftr'>© copyright Group-4 CSCI-5030 2021</div>
                 </div>
                         <!--Java Script-->
                         <script>
+						
+						<!-- Script for Multi-Lingual Frontend -->
```
Comment (MarshallOkafor) [inline index.html:63]: Awesome! I can see that this is the JavaScript code that will switch the HTML page between English and Irish Language. Nice!

**Review by MarshallOkafor — APPROVED** [review]
Well done Harry, I saw that you added some JavaScript codes and did some modifications in the HTML file. 

This is to enable the switch between English and Irish Language. I also tested it and everything works just fine.

I was also thinking, dont we need to translate the header and the footer of the page too?

**Pull request merged**